frontend/packages/operator-lifecycle-manager/src/components/k8s-resource.tsx

```diff
@@ -49,10 +49,10 @@ const DEFAULT_RESOURCES: CRDDescription['resources'] = [
 ];
 
 const tableColumnClasses = [
-  classNames('col-lg-4', 'col-md-4', 'col-sm-4', 'col-xs-6'),
-  classNames('col-lg-2', 'col-md-2', 'col-sm-4', 'col-xs-6'),
-  classNames('col-lg-2', 'hidden-md', 'hidden-sm', 'hidden-xs'),
-  classNames('col-lg-4', 'col-md-4', 'col-sm-4', 'hidden-xs'),
+  classNames(''),
+  classNames('', 'pf-u-w-16-on-md'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-lg', 'pf-u-w-16-on-lg'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm'),
```

Comment:
As christian noted, we dont need classNames here as well and also in other file (subscription.tsx).

Reply:
@karthikjeeyar I tried to keep it as-is, not add classNames to files where it wasn't pre-existing and keep files where classNames were pre-existing..

Neway, I removed classNames here and the other file as well